src/diamond/metric.py

```diff
@@ -24,7 +30,7 @@ def __init__(self, path, value, raw_value=None, timestamp=None, precision=0,
         """
 
         # Validate the path, value and metric_type submitted
-        if (None in [path, value] or metric_type not in self._METRIC_TYPES):
+        if (None in [path, value] or metric_type not in ['COUNTER', 'GAUGE']):
```

Comment:
```
>>> dis.dis(lambda: ['a','b'])
  1           0 LOAD_CONST               1 ('a')
              3 LOAD_CONST               2 ('b')
              6 BUILD_LIST               2
              9 RETURN_VALUE
>>> dis.dis(lambda: ('a','b'))
  1           0 LOAD_CONST               3 (('a', 'b'))
              3 RETURN_VALUE
```

Defining a tuple is an O(1), but a list is O(n). Minor in this case, just good hygiene.